app/models/shift.rb

```diff
@@ -31,21 +32,33 @@ def can_be_dropped?
   end
 
   private
+  def any_shift_time_blank?
+    return if self.shift_start.blank? || self.shift_end.blank?
```

Comment:
So here Gabrielle, you want the following:

```suggestion
    self.shift_start.blank? || self.shift_end.blank?
```